### ISSUE TITLE
Old minor version `v18.4.0` in `javascript-node:0-18-bullseye` image

### ISSUE THREAD

**Issue by fwilhe2** [microsoft/vscode-dev-containers#1590]
Hi,

I'm building something where I want to use latest and greatest nodejs features, and I found that the mcr.microsoft.com/vscode/devcontainers/javascript-node:0-18-bullseye image as of today does not include the latest minor version `v18.7.0`.

I would expect to get the latest minor version, is there a reason for shipping older versions? Are there other tags which provide the latest version?

```
$ docker pull mcr.microsoft.com/vscode/devcontainers/javascript-node:0-18-bullseye
0-18-bullseye: Pulling from vscode/devcontainers/javascript-node
1339eaac5b67: Pull complete 
4c78fa1b9799: Pull complete 
14f0d2bd5243: Pull complete 
76e5964a957d: Pull complete 
cc4bb1a04a94: Pull complete 
eba57464a96a: Pull complete 
1c6bdae10107: Pull complete 
93c6143f0972: Pull complete 
5681e060b2f8: Pull complete 
bba7bda38070: Pull complete 
b298b78254f4: Pull complete 
Digest: sha256:28d382643a2f24b421c5ae65661cb8dec3f4735fdb7427be1634d20aa12466f4
Status: Downloaded newer image for mcr.microsoft.com/vscode/devcontainers/javascript-node:0-18-bullseye
mcr.microsoft.com/vscode/devcontainers/javascript-node:0-18-bullseye

$ docker run -it --rm mcr.microsoft.com/vscode/devcontainers/javascript-node:0-18-bullseye
Welcome to Node.js v18.4.0.
Type ".help" for more information.
> 
```

The latest image seems to be built 7 weeks ago:

```
mcr.microsoft.com/vscode/devcontainers/javascript-node       0-18-bullseye     36132f8736ca   7 weeks ago      1.13GB
```

**Comment by joshspicer** [comment]
Thanks for the note.  I'm going to trigger a re-release of our images to get the latest patches published.

**Comment by joshspicer** [comment]
These will be available under repo release v0.245.2

**Comment by fwilhe2** [comment]
Thanks! Can verify :)
